### PR TITLE
[Portal Generated] Review request for Microsoft.Batch to add version 2018-12-01

### PR DESCRIPTION
**Pull request by kiwidev**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/kiwidev/azure-rest-api-specs/dev-batch-Microsoft.Batch-2018-12-01).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/batch/resource-manager/Microsoft.Batch/stable/2018-12-01/BatchManagement.json [hunk 1]
```diff
@@ -0,0 +1,4019 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation was successful. The response contains the pool entity.",
+            "schema": {
+              "$ref": "#/definitions/Pool"
+            },
+            "headers": {
+              "ETag": {
+                "description": "The ETag HTTP response header. This is an opaque string. You can use i
```
Comment (nschonni) [inline specification/batch/resource-manager/Microsoft.Batch/stable/2018-12-01/BatchManagement.json]: visibile -> visible

Comment (nschonni) [inline specification/batch/resource-manager/Microsoft.Batch/stable/2018-12-01/BatchManagement.json]: visibile -> visible

Comment (nschonni) [inline specification/batch/resource-manager/Microsoft.Batch/stable/2018-12-01/BatchManagement.json]: VNETs -> VNets

**Review by nschonni — COMMENTED** [review]
Few things I cleaned up in the other versions already

**Comment by kiwidev** [comment]
Thanks @nschonni, have resolved.

**Comment by kiwidev** [comment]
FYI @matthchr, @xingwu1

**Review by xingwu1 — APPROVED** [review]

**Comment by xingwu1** [comment]
Do not merge until we finish the deployment.

**Comment by jhendrixMSFT** [comment]
@kiwidev has this been reviewed by ARM?

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
There are a few examples that need fixing, please see the [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/463028366) for more info.

**Comment by jhendrixMSFT** [comment]
@kiwidev any update?

**Comment by xingwu1** [comment]
@jhendrixMSFT Service is rolling out. You can merge it now.

**Comment by xingwu1** [comment]
@jhendrixMSFT There is a field named "password" which is required for POST. We added this field to the "required" section. However, it will not send back by service when you call GET. Since there is no "writeOnly" supporting for autorest now, I don't know what can we do for the examples.

**Comment by jhendrixMSFT** [comment]
@xingwu1 has ARM reviewed this API version?

**Comment by kiwidev** [comment]
@jhendrixMSFT waiting on ARM review. Also waiting for service API version to be enabled, so keeping the "Do Not Merge" label

**Comment by ravbhatnagar** [comment]
Signing off from ARM side.

**Comment by jhendrixMSFT** [comment]
@kiwidev please don't forget to look at the model validation failures.

**Comment by kiwidev** [comment]
@jhendrixMSFT I'm not seeing any model validation failures that aren't covered by suppressions. The OpenAPI Hub portal doesn't show any model validation issues. Can you highlight any that are an issue? Thanks

**Comment by jhendrixMSFT** [comment]
Thanks for clarifying, I see these were suppressed now although it's odd the task shows as failed but there are no errors, see the [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/466686515).  Will sign off.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**